zosia16/urls.py

```diff
@@ -24,4 +26,4 @@
     url(r'^blog/', include('blog.urls')),
     url(r'^sponsors/', include('sponsors.urls')),
     url(r'^lectures/', include('lectures.urls')),
-]
+] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
Can you double check it doesn't serve static when DEBUG=False? And add a note if it really does not.